idaplugin/rematch/network.py

```diff
@@ -111,15 +115,20 @@ def query(method, url, server=None, token=None, params=None, json=False):
 def get_server(server):
   """getting and finalzing server address"""
 
-  if not server:
-    if 'server' not in config or not config['server']:
-      raise exceptions.QueryException()
-    server = config['server']
-  if not (server.startswith("http://") or server.startswith("http://")):
-    server = "http://" + server
-  if not server.endswith("/"):
-    server = server + "/"
-
+  try:
+    if not server:
+      if 'server' not in config and not config['login']['server']:
+        raise exceptions.QueryException()
+      server = config['login']['server']
+    if not (server.startswith("http://") or server.startswith("http://")):
+      server = "http://" + server
+    if not server.endswith("/"):
+      server = server + "/"
+  except:
+    import traceback
+    import sys
+    traceback.print_exc()
+    traceback.print_exception(*sys.exc_info())
```

Comment:
If you want to log this warning I think it better conform to how we log stuff in other places, using the logger API. However this doesn't seem like a major error and an hardcoded default value seems like a simpler approach. Up to you.

also IIRC `traceback.print_exc()` and `traceback.print_exception(*sys.exc_info())` do the same thing.